### PR TITLE
Pam 304 link updates

### PR DESCRIPTION
**Pull request by pamgoodrich**
Simplified steps for adding markdown links to articles. Added info about linking to endpoints in the API description from within the same file. Struggled a bit with code snippets in steps. There may be a rendering bug here.

### CODE REVIEW

File: 4.-documentation/linking-between-articles.md [hunk 1]
```diff
@@ -11,48 +11,36 @@ Add hyperlinks to other Markdown files or API elements to your documentation fil
...
 
-5. Append the HTTP request method to the end of the URL. Example:
-
-   `[creating a verification](../reference/your-api-file.yaml/paths/~1verification-requests/post)`
+1. From the **APIs** tab, open the API you are linking to, and then right-click the endpoint you want to link to.
+2. Select **Copy Path**.
+3. Paste the path into your markdown file, using this format:
+        `[Display text](reference/-your-api-file.yaml/paths/~1endpoint-name)`.
+4. Append the HTTP request method to the end of the URL. Example:
+        `[creating a verification](../reference/your-api-file.yaml/paths/~1verification-requests/post)`.
```
Comment (heitortsergent) [inline 4.-documentation/linking-between-articles.md:31]: I just tested this, and I think we could add an additional step here. The user needs to append `../` to the path they copy for the link to work, so if the "Copy Path" is:

`reference/-your-api-file.yaml/paths/~1endpoint-name`

Maybe in step 4 we can say "Append `../` to the path you copied:

`../reference/-your-api-file.yaml/paths/~1endpoint-name`

Comment (heitortsergent) [reply 4.-documentation/linking-between-articles.md:31]: And maybe we can keep [Display text] as the text in step 3 and 4?

File: 4.-documentation/linking-between-articles.md [hunk 2]
```diff
@@ -11,48 +11,36 @@ Add hyperlinks to other Markdown files or API elements to your documentation fil
...
 
 > **Note**: The copied path does not include the HTTP request method. You will need to append that yourself.
 
+To add operations links to API descriptions in the same specification file, use the encoded JSON path. Example:
+        `[link text](paths/~1users~1{userId}/get`
```
Comment (heitortsergent) [inline 4.-documentation/linking-between-articles.md:36]: This is missing a `)` after `get`.

Also I tried this and starting the link with `paths/...` didn't work for me, not sure I'm doing something wrong. I had to copy the relative path to an endpoint, so it included the API name before path like: `API-spec.yaml/paths/~1users~1{userId}/get`

**Review by heitortsergent — COMMENTED** [review]
@pamgoodrich I just reviewed it, left some comments based on my testing of following the steps.

File: 4.-documentation/linking-between-articles.md [hunk 3]
```diff
@@ -11,48 +11,36 @@ Add hyperlinks to other Markdown files or API elements to your documentation fil
...
-
 2. Select **Copy Path**. 
 
 ![link-api-overview.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/Rr0cCDrLpB0)
 
 3. Open the **Markdown file** you want to add links to.
-
 4. Add the link in this format:
-
    `[Display text](reference/your-api-file.yaml)`
```
Comment (heitortsergent) [inline 4.-documentation/linking-between-articles.md:20]: Does this also need to be updated to append `../` to `reference/your-api-file.yaml`?

File: 4.-documentation/linking-between-articles.md (hunk 1 above)
Comment (pamgoodrich) [reply 4.-documentation/linking-between-articles.md:31]: Done

**Review by heitortsergent — APPROVED** [review]
Looks good to me 👍

**Pull request merged**